src/cart/crt_internal.h

```diff
@@ -71,4 +71,17 @@
 
 extern uint32_t crt_swim_rpc_timeout;
 
+#undef CRT_DEBUG_TRACE
```

Comment:
One comment here...you could potentially add this when BUILD_TYPE == debug

Reply:
Removed.  should not be there.